src/components/icons/FeelOptional.svg

```diff
@@ -1 +1 @@
-<svg viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M5.845 7.04V5.93h4.307v1.11H5.845Zm0 3.07V9h4.307v1.11H5.845Z" fill="#505562"/><path fill-rule="evenodd" clip-rule="evenodd" d="M3.286 8a4.714 4.714 0 1 0 9.428 0 4.714 4.714 0 0 0-9.428 0ZM8 2a6 6 0 1 0 0 12A6 6 0 0 0 8 2Z" fill="#505562"/></svg>
\ No newline at end of file
+<svg viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M5.845 7.04V5.93h4.307v1.11H5.845Zm0 3.07V9h4.307v1.11H5.845Z" fill="var(--svg-stroke-color)"/><path fill-rule="evenodd" clip-rule="evenodd" d="M3.286 8a4.714 4.714 0 1 0 9.428 0 4.714 4.714 0 0 0-9.428 0ZM8 2a6 6 0 1 0 0 12A6 6 0 0 0 8 2Z" fill="var(--svg-stroke-color)"/></svg>
```

Comment:
Let us use `fill="currentColor"` instead and color via parent (+CSS).

Reply:
Forgot that `currentColor` exists for SVG. Changed 👍